plugins/woocommerce/tests/legacy/unit-tests/importer/product.php

```
@@ -659,12 +667,98 @@ public function test_get_parsed_data() {
+		$expected_brands = array(
+			array( 'TopBrand', 'KidCakes' ),                  // Woo Logo: "TopBrand, TopBrand > KidCakes".
+			array( 'Slice', 'TopBrand' ),                     // Woo Album #1: "TopBrand > Slice, TopBrand".
+			array( 'Another Brand' ),                         // WooCommerce Product CSV Suite: "Another Brand".
+			array( 'TopBrand', 'KidCakes' ),                  // Ship Your Idea: "TopBrand, TopBrand > KidCakes".
+			array(),                                          // Variation 1: No brands.
+			array(),                                          // Variation 2: No brands.
+			array( 'TopBrand', 'KidCakes', 'Slice' ),         // Best Woo Products: "TopBrand, TopBrand > KidCakes, TopBrand > Slice".
+		);
+
+		$importer = new WC_Product_CSV_Importer( $this->csv_file, $args );
+		$import = $importer->import();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$import'. (undefined)

(UnusedLocalVariable)